### ISSUE TITLE
다른 modality의 pretrained model / dataset 사용이 가능한가요?

### ISSUE THREAD

**Issue by MilkClouds** [LISTatSNU/FastMRI_challenge#174]
예를 들어 rgb image에 대한 pretrained weights나 외부 dataset 사용이 가능한지, modality 상관없이 모든 외부 모델이나 데이터셋 사용이 불가능한 것인지 궁금합니다.

**Comment by MilkClouds** [comment]
그리고 모델 크기와 관련하여, 이전 8GB VRAM의 제한이 있었던 것으로 보이는데, 정확히 규정이 어떻게 되는 것일까요?
gradient checkpointing 등의 테크닉으로 모델을 잘라 학습하는 것도, 아니면 여러 모델을 sequantial하게 학습하는 것도 가능할텐데 모델 파라미터 수에 제한이 있다거나 그런 것인가요?

**Comment by wogur110** [comment]
안녕하세요, 2023 SNU FastMRI Challenge 조교 배재혁입니다.

이번 대회에서는 modality에 상관 없이 모든 외부 모델이나 데이터셋 사용이 불가능합니다.
또한, 작년과 마찬가지로 모델 크기는 8GB VRAM의 제한이 있습니다. 즉, 제공해드린 Vessl GPU 서버에서 돌아갈 수 있어야 합니다. 말씀해주신대로, 모델을 잘라 학습하거나 여러 모델을 sequential 하게 학습하더라도 Vessl GPU 서버에서 돌아간다면 규정에 어긋나지 않습니다. 이 부분에 대해서 메일로 재공지해드리겠습니다.

감사합니다.

**Comment by junjinyong** [comment]
8GB 제한이 training 시에 적용되는 것인가요? 아니면 inference 시에 적용되는 것인가요?

**Comment by junjinyong** [comment]
@wogur110 

8GB 제한이 training 시에 적용되는 것인가요? 아니면 inference 시에 적용되는 것인가요?

**Comment by wogur110** [comment]
training과 inference 과정 모두 Vessl 서버 내에서 돌아가야 하므로 8GB로 제한됩니다.
